blockdevice/stats.go

```diff
@@ -474,3 +475,13 @@ func (fs FS) SysBlockDeviceUnderlyingDevices(device string) (UnderlyingDeviceInf
 	return UnderlyingDeviceInfo{DeviceNames: underlying}, nil
 
 }
+
+// SysBlockDeviceSizeBytes returns the size of the block device from /sys/block/<device>/size
+// in bytes by multiplying the value by the Linux sector length of 512.
+func (fs FS) SysBlockDeviceSizeBytes(device string) (uint64, error) {
+	size, err := util.ReadUintFromFile(fs.sys.Path(sysBlockPath, device, sysBlockSize))
+	if err != nil {
+		return 0, err
+	}
+	return 512 * size, nil
```

Comment:
Let's make this a const SectorSize

Reply:
https://github.com/prometheus/procfs/pull/650/commits/b70d052f2b0a97e986fedba7bd6260931f3e31f6